kernel/configs/x86_64_kata_kvm_4.14.x

```diff
@@ -2256,6 +2269,7 @@ CONFIG_HUGETLBFS=y
 CONFIG_HUGETLB_PAGE=y
 CONFIG_ARCH_HAS_GIGANTIC_PAGE=y
 # CONFIG_CONFIGFS_FS is not set
+CONFIG_EFIVAR_FS=m
```

Comment:
If we need this at boot time, I suggest to make it build-in.

Reply:
btw, I think is not needed  so we should try to remove it

Reply:
Agree. In any case we dont really support loadable modules.

Reply:
Yeah I agree guys, I didn't pay attention when I applied default config options.